classroom.rb

```diff
@@ -48,4 +60,8 @@ def belongs_to_person
     parent_permission = @person_data.parent_permission
     new.Person(age, name, parent_permission)
   end
+
+  def rental_list
+    @@rentals
+  end
```

Comment:
- [ ] Kindly move to the App class or other appropriate place. It is not a behavior of a single Rental object and as such it doesn't belong here.